README.md

```diff
@@ -112,6 +112,41 @@ java $JAVA_OPTS \
      -jar $JETTY_HOME/start.jar \
      "$@"
 ```
+### Extending logging
+The `java.util.logging` configuration may be changed at runtime by providing an alternate
+properties file. This can be done either by extending the image and replacing the
+default configuration at `$JETTY_BASE/etc/java-util-logging.properties`, or a new
+configuration can be provided as part of the application (eg `WEB-INF/logging.properties`)
+and setting the Java System Property `java.util.logging.config.file` to point to it.
+System properties can be set by setting the unix environment variable `JAVA_USER_OPTS`.
+
+An example of running the image locally with a mounted application is:
+```bash
+docker run \
+  -e JAVA_USER_OPTS=-Djava.util.logging.config.file=WEB-INF/logging.properties \
+  -v /some-path/your-application:/app gcr.io/google_appengine/jetty
+```
+
+When deploying via the Cloud SDK and/or plugin, the `JAVA_USER_OPTS` may be set in
+the `app.yaml` file:
+```yaml
+env_variables:
+  JAVA_USER_OPTS: -Djava.util.logging.config.file=WEB-INF/logging.properties
+```
+
+The default logging.properties file contains the following to configure `java.util.logging`
+to use the gcloud stackdriver logging mechanism:
+```
+.level=INFO
+
+handlers=com.google.cloud.logging.LoggingHandler
+com.google.cloud.logging.LoggingHandler.level=FINE
```

Comment:
I don't think so. This just says it will log any FINE entries that are generated, but the line 140 sets the default generation to just be INFO.    By leaving the FINE in the handler it means that you only need to change the level in one place.   

In fact there is an argument to change it to FINEST